pinot-segment-local/src/test/java/org/apache/pinot/segment/local/segment/store/TextIndexUtilsTest.java

```diff
@@ -45,7 +45,8 @@ public void testRoundTripProperties()
 
     TextIndexUtils.writeConfigToPropertiesFile(TEMP_DIR, config);
     TextIndexConfig readConfig = TextIndexUtils.getUpdatedConfigFromPropertiesFile(
-        new File(TEMP_DIR, V1Constants.Indexes.LUCENE_TEXT_INDEX_PROPERTIES_FILE), config);
+        new File(TEMP_DIR, V1Constants.Indexes.LUCENE_TEXT_INDEX_PROPERTIES_FILE),
+        JsonUtils.stringToObject("{}", TextIndexConfig.class));
```

Comment:
Do we need a separate test for this?

Reply:
Earlier test was writing the config to property file and then passing the same to merge it with the property file which I thought may not be testing much, hence updated the same test by passing an empty config to merge it with the property file.